Jenkinsfile

```diff
@@ -17,25 +16,89 @@ node('vetsgov-general-purpose') {
   }
 
   def commonStages = load "vets-website/jenkins/common.groovy"
+  def envUsedCache = [:]
 
   // setupStage
   dockerContainer = commonStages.setup()
 
-  stage('Lint|Security|Unit') {
-    if (params.cmsEnvBuildOverride != 'none') { return }
+  stage('Main') {
+    def contentOnlyBuild = params.cmsEnvBuildOverride != 'none'
+    def assetSource = contentOnlyBuild ? ref : 'local'
 
     try {
       parallel (
         failFast: true,
 
+        buildDev: {
```

Comment:
For each env, I basically copied the [contents of the loop in the `buildAll` function from `common.groovy`](https://github.com/department-of-veterans-affairs/vets-website/blob/fa55666af5dc82d6c84258d33812bf636d14e23d/jenkins/common.groovy#L282-L300). 

The only difference between each environment is the environment name. 

I tried to extract the common code into a helper function that accepted the environment name as a parameter in https://github.com/department-of-veterans-affairs/vets-website/pull/16965/commits/ec1822f4138ceeeba2d3bf4a8ff3ca873ba9b115, but I encountered a weird error, and I didn't want to spend too much time n this experiment.